src/main/resources/application.conf

```diff
@@ -548,6 +548,44 @@ mantis {
     }
   }
 
+  faucet {
```

Comment:
I suggest moving this outside of mantis section (as in `universal/conf/faucet.conf`) and into a separate file